### PR TITLE
Add bindings for Start/Stop operation

### PR DESCRIPTION
**Pull request by NickKeller**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -352,3 +352,24 @@ def test_aks_disable_addons_confcom_addon(self, resource_group, resource_group_l
...
+            'name': aks_name
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name}'
+        self.cmd(create_cmd, checks=[
+            self.check('provisioningState', 'Succeeded'),
+        ])
+
+        stop_cmd = 'aks stop --resource-group={resource_group} --name={name}'
+        self.cmd(stop_cmd)
```
Comment (marwanad) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:372]: should we add some validation here on the output?

**Comment by yonzhan** [comment]
aks

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py [hunk 2]
```diff
@@ -352,3 +352,24 @@ def test_aks_disable_addons_confcom_addon(self, resource_group, resource_group_l
             self.check('addonProfiles.accsgxdeviceplugin.enabled', False),
             self.check('addonProfiles.accsgxdeviceplugin.config', None)
         ])
+
+    @live_only()
```
Comment (arrownj) [inline src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:356]: Is there a special reason to make it `live_only` ?

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py (hunk 1 above)
Comment (NickKeller) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:372]: The only validation to be done is the status code, which is already done by `self.cmd`

File: src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py (hunk 2 above)
Comment (NickKeller) [reply src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:356]: It needs an active cluster to start and stop

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @NickKeller is this PR ready to merge?

**Pull request merged**